### ISSUE TITLE
Support workspaces in release workflow

### ISSUE THREAD

**Issue by benfrankel** [TheBevyFlock/bevy_quickstart#194]
Provide an `env` variable (off by default) to specify a workspace sub-directory so that the release workflow can use that and Just Work.

**Comment by benfrankel** [comment]
Currently this _should_ Just Work as long as you:
1. Set `assets_path` to `path/to/assets` if needed.
2. Keep `Trunk.toml` in the root directory, with `build.target` pointing to `path/to/web/index.html` if needed.
3. Make sure `cargo build` with no args will build the "main app" that you want to release. This can be done by setting the main app as the root package or one of the `default-members`, and setting `cargo_build_binary_name` if needed.

This should be explained in the `workflows.md` documentation.

**Comment by janhohenheim** [comment]
Anything left here?

**Comment by benfrankel** [comment]
It seems like the `cargo_build_package` env variable approach to improve workspace support should be unblocked now that we've migrated to Bevy CLI. Although I haven't actually tested that workspaces are compatible with Bevy CLI yet.

Tl;dr there's some more work to do here, but I don't use workspaces personally so it's low priority for me.

**Comment by janhohenheim** [comment]
Update: `avian`, which uses workspaces, required setting `cache-all-crates: "true"` in the CI to not recompile everything. Not sure if that's an `avian` quirk or a workspace quirk.

**Comment by benfrankel** [comment]
By "recompile everything" do you mean all the crates in the workspace, or all the transitive dependencies too?

**Comment by janhohenheim** [comment]
transitives too, for some reason
